modules/templates/util_render.go

```diff
@@ -143,7 +143,7 @@ func RenderLabel(ctx context.Context, label *issues_model.Label) template.HTML {
 
 	if labelScope == "" {
 		// Regular label
-		s := fmt.Sprintf("<div class='ui label' style='color: %s !important; background-color: %s !important' title='%s'>%s</div>",
+		s := fmt.Sprintf("<div class='ui label' style='color: %s !important; background-color: %s !important' data-tooltip-content='%s'>%s</div>",
```

Comment:
Could it instead do this so it will render as a tooltip, but fallback to a native browser title when JavaScript is not loaded, since data-tooltip-content gets loaded from title automatically if it is present and empty?

```suggestion
		s := fmt.Sprintf("<div class='ui label' style='color: %s !important; background-color: %s !important' data-tooltip-content title='%s'>%s</div>",
```